packages/bot/src/app.jsx

```diff
@@ -1,9 +1,10 @@
 
-import { Provider } from 'mobx-react';
-import React        from 'react';
-// import Button       from 'deriv-components/lib/button'; // eslint-disable-line import/no-unresolved
-import Bot          from './components/bot.jsx';
-import RootStore    from './stores';
+// import Button                from 'deriv-components/lib/button'; // eslint-disable-line import/no-unresolved
```

Comment:
@aaron-binary i think this can be removed. it was for test. we dont need it anymore